src/app/dashboard-view/user.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface Users {
```

Comment:
```suggestion
export interface User {
```
You are creating the interface for a single `User`. If there are multiple users you would define it as an array of User.